test/test-ApplyVisitorTest.cc

```diff
@@ -88,14 +85,16 @@ TEST_CASE("It should invoke Functor for member of container elements", "[Visitor
     Functor f;
     refract::Visitor v(f);
 
-    refract::IElement* e
-        = Build(new ArrayElement)(IElement::Create(3))(IElement::Create(false))(IElement::Create("Ehlo"));
+    auto e = make_element<ArrayElement>();
+    auto& content = e->get();
+    content.push_back(make_primitive(3.0));
+    content.push_back(make_primitive(false));
+    content.push_back(make_primitive("Ehlo"));
+
     v.visit(*e);
 
     REQUIRE(f.GCounter == 1); // just array
     REQUIRE(f.SCounter == 0);
-
-    delete e;
 }
 
 TEST_CASE("It should recognize Element Type by `Is` type operand", "[Visitor]")
```

Comment:
By blame, my shame :)
We should implement or remove it completly
test report non-existent behavior....

Reply:
What do you mean?

Reply:
There are tests with body commented out by `#if 0` 
we should (sorted by prefered solution :)
- make implementation
- remove it from tests
- commentout full test not just body

But for this branche probably will be most effective just comment out related test

We do not want to have repoting functionality of test which is not implemeted